### PR TITLE
Report scenarios that pass on retry to github

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Report scenarios that pass on retry to github.

## Related Issue

- Related https://github.com/owncloud/web/issues/5257 

## Motivation and Context

There are many intermittent failing tests that fail occasionally in drone but we never know about them because they pass on retry. With this PR we will at least know about scenarios that pass on retry

## How Has This Been Tested?

🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview5</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16437/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>
<details><summary>:boom: Acceptance tests <strong>preview5</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16437/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>
<details><summary>:boom: Acceptance tests <strong>preview2</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview6</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview4</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview8</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview7</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview9</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview5</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16522/

<p>


### webUIPreview-imageMediaViewer-feature-13.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-15.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview8</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16525/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview6</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16525/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16681/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>preview5</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16681/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140
<details><summary>:boom: Acceptance tests <strong>preview8</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16684/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140
<details><summary>:boom: Acceptance tests <strong>preview3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16703/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140
<details><summary>:boom: Acceptance tests <strong>preview5</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16710/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
%s
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140
<details><summary>:boom: Acceptance tests <strong>preview1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16710/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>preview4</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview6</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview8</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview9</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview1</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview5</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview7</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>preview2</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140
<details><summary>:boom: Acceptance tests <strong>preview2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16721/

<p>


### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-67.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>preview3</strong>
The following scenarios passed on retry:
webUIPreview/imageMediaViewer.feature:67
webUIPreview/imageMediaViewer.feature:83
webUIPreview/imageMediaViewer.feature:90
webUIPreview/imageMediaViewer.feature:158
webUIPreview/imageMediaViewer.feature:140

**Comment by ownclouders** [comment]
Results for <strong>oCISFavorites</strong>
The following scenarios passed on retry:
- webUIFavorites/unfavoriteFile.feature:72
- webUIFavorites/favoritesFile.feature:58
- webUIFavorites/favoritesFile.feature:46
- webUIFavorites/favoritesFile.feature:110
- webUIFavorites/favoritesFile.feature:30
- webUIFavorites/favoritesFile.feature:131
- webUIFavorites/unfavoriteFile.feature:55
- webUIFavorites/unfavoriteFile.feature:104
- webUIFavorites/unfavoriteFile.feature:34
- webUIFavorites/favoritesFile.feature:14
- webUIFavorites/favoritesFile.feature:74
- webUIFavorites/unfavoriteFile.feature:13
- webUIFavorites/favoritesFile.feature:80
- webUIFavorites/unfavoriteFile.feature:89
- webUIFavorites/favoritesFile.feature:85
- webUIFavorites/favoritesFile.feature:65

**Comment by ownclouders** [comment]
Results for <strong>oCISFavorites</strong>
The following scenarios passed on retry:
- webUIFavorites/unfavoriteFile.feature:72
- webUIFavorites/favoritesFile.feature:58
- webUIFavorites/favoritesFile.feature:46
- webUIFavorites/favoritesFile.feature:110
- webUIFavorites/favoritesFile.feature:30
- webUIFavorites/favoritesFile.feature:131
- webUIFavorites/unfavoriteFile.feature:55
- webUIFavorites/unfavoriteFile.feature:104
- webUIFavorites/unfavoriteFile.feature:34
- webUIFavorites/favoritesFile.feature:14
- webUIFavorites/favoritesFile.feature:74
- webUIFavorites/unfavoriteFile.feature:13
- webUIFavorites/favoritesFile.feature:80
- webUIFavorites/unfavoriteFile.feature:89
- webUIFavorites/favoritesFile.feature:85
- webUIFavorites/favoritesFile.feature:65

**Comment by pascalwengerter** [comment]
> Results for **oCISFavorites**
> The following scenarios passed on retry:
> 
>     * webUIFavorites/unfavoriteFile.feature:72
> 
>     * webUIFavorites/favoritesFile.feature:58
> 
>     * webUIFavorites/favoritesFile.feature:46
> 
>     * webUIFavorites/favoritesFile.feature:110
> 
>     * webUIFavorites/favoritesFile.feature:30
> 
>     * webUIFavorites/favoritesFile.feature:131
> 
>     * webUIFavorites/unfavoriteFile.feature:55
> 
>     * webUIFavorites/unfavoriteFile.feature:104
> 
>     * webUIFavorites/unfavoriteFile.feature:34
> 
>     * webUIFavorites/favoritesFile.feature:14
> 
>     * webUIFavorites/favoritesFile.feature:74
> 
>     * webUIFavorites/unfavoriteFile.feature:13
> 
>     * webUIFavorites/favoritesFile.feature:80
> 
>     * webUIFavorites/unfavoriteFile.feature:89
> 
>     * webUIFavorites/favoritesFile.feature:85
> 
>     * webUIFavorites/favoritesFile.feature:65

Not sure what's getting tested here but this is a confusing message since favorites don't work in oCIS

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong>
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:353
:boom: Acceptance tests pipeline <strong>oC10SharingExternalRoot</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/web/16744/38/1

**Comment by dpakach** [comment]
> Not sure what's getting tested here but this is a confusing message since favorites don't work in oCIS

@pascalwengerter  its a bug in the test runner script I'm trying to fix that, this PR is still WIP

**Comment by ownclouders** [comment]
Results for <strong>oCISFavorites</strong>
:boom: Acceptance tests pipeline <strong>oCISFavorites</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/web/16763/48/1

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPermissions1</strong>
:boom: Acceptance tests pipeline <strong>oCISSharingPermissions1</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/web/16767/54/1

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFolderPermissions</strong>
:boom: Acceptance tests pipeline <strong>oC10SharingFolderPermissions</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/web/16767/19/1

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong>
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

File: .drone.star [hunk 1]
```diff
@@ -2450,7 +2450,8 @@ def buildGithubCommentForBuildStopped(suite, alternateSuiteName):
         },
     }]
 
-def githubComment():
+def githubComment(alternateSuiteName):
+    prefix = "Results for <strong>%s</strong>" % alternateSuiteName
```
Comment (phil-davis) [inline .drone.star:2454]: IMO this is going to also prefix the other failure comments, like ":boom: Acceptance tests pipeline <strong>%s</strong> failed. The build has been cancelled."

And those already have `alternateSuiteName` embedded in the text.

But we can can merge this - it does not break anything - and then cleanup whichever comments have duplication.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**